### ISSUE TITLE
Storing the same certificate more than once creates more than one entry in the database

### ISSUE THREAD

**Issue by jer8me** [jer8me/CertStore#1]
When using the `store` command to store a certificate, running the command multiple times with the same certificate file creates a new entry in the database for the same certificate.
For example:

```bash
$ ./CertStore.exe store google.pem --dbpass $DB_PASSWORD
Certificate google.pem successfully stored (certificate ID=6)
```
```bash
$ ./CertStore.exe store google.pem --dbpass $DB_PASSWORD
Certificate google.pem successfully stored (certificate ID=7)
```
We should not create a new entry for the same content. The command should succeed and return the ID of the existing certificate.

**Comment by jer8me** [comment]
This issue is fixed at a999db9739f01730bf61dc40fee68758f782a8e3

**Comment by jer8me** [comment]
This issue is fixed at a999db9739f01730bf61dc40fee68758f782a8e3
